docs/pages/database-access/guides/rds-proxy.mdx

```diff
@@ -0,0 +1,201 @@
+---
+title: Database Access with AWS RDS Proxy
+description: How to configure Teleport Database Access with AWS RDS Proxy.
+---
+
+This guide will help you to set up Teleport to access your AWS RDS proxies.
```

Comment:
```suggestion
This guide will help you to set up Teleport to access your AWS RDS Proxy instances.
```
It looks like the AWS docs say "RDS Proxy instance", instead of "RDS Proxies" (https://aws.amazon.com/rds/proxy/)